### ISSUE TITLE
How to define batch norm layer? Does the method work on it?

### ISSUE THREAD

**Issue by anerudhr** [deepmind/deep-verify#3]
Hi all, I was wondering how one could add a batch normalization layer to the code. Also, can the method work with batch normalization?

**Comment by anerudhr** [comment]
It does support batch normalization. You just need to add a line to layers() function in verify.py as "('batch_normalization')". Hope this is helpful to people.
